modules/luci-mod-admin-full/luasrc/model/cbi/admin_system/leds.lua

```diff
@@ -144,11 +138,11 @@ for p in nixio.fs.glob("/sys/bus/usb/devices/[0-9]*/manufacturer") do
 	usbdev:value(id, "%s (%s - %s)" %{ id, mf, pr })
 end
 
-for p in nixio.fs.glob("/sys/bus/usb/devices/*/usb[0-9]*-port[0-9]*") do
-	local bus, port = p:match("usb(%d+)-port(%d+)")
-	if bus and port then
-		usbport:value("usb%u-port%u" %{ tonumber(bus), tonumber(port) },
-		              "Hub %u, Port %u" %{ tonumber(bus), tonumber(port) })
+for p in nixio.fs.glob("/sys/bus/usb/devices/*/*-port[0-9]*") do
+	local port = p:match("/sys/bus/usb/devices/.*/(.*-port%d+)$")
+	if port then
+		usbport:value("%s" %{ port },
+		              "%s %s" %{ port, nixio.fs.stat("%s/device" %{p}) ~= nil and "(device attached)" or ""})
```

Comment:
Please add helper variable a few lines above assigning the device presence state and make the string translatable:

```
local attached = nixio.fs.stat("%s/device" % p) and ' (%s)' % translate('device attached') or ''
usbport:value(port, port .. attached)
```

Reply:
done